src/controllers/FeedController.js

```diff
@@ -89,9 +92,23 @@ module.exports = {
 
       const result = await getFeedByIdService.execute({ id });
 
-      res.status(200).json(result);
+      return res.status(200).json(result);
+    } catch (err) {
+      return next(err);
+    }
+  },
+
+  async approveFeeds(req, res, next) {
+    try {
+      const { id } = req.params;
+
+      const { is_pendent } = req.body;
+
```

Comment:
@gumeeee creio que não é necessário pegar o **is_pendent** do **body** da requisição, pois quando essa rota for executada sempre será uma ação de aprovar o feed, então você poder definir **is_pendent** igual true na hora que for atualizar o dados.